common/vpcclient/client/payload/body_provider.go

```diff
@@ -96,6 +97,6 @@ func (p *MultipartFileBody) copyBody() {
 
 // Close ...
 func (p *MultipartFileBody) Close() {
-	p.multipartWriter.Close()
-	p.pipeWriter.Close()
+	p.multipartWriter.Close() //nolint:gosec
```

Comment:
what is the solution to fix the security issues?

Reply:
Close returns error. Which we have not handled here. I thought of doing err := p.multipartWriter.Close() and check if err == nil But i don't know know what to do in case of err is not nil. So ignored the warning.